app/ios/Podfile.lock

```diff
@@ -1,21 +1,33 @@
 PODS:
   - Flutter (1.0.0)
+  - flutter_full_pdf_viewer (1.0.1):
+    - Flutter
   - integration_test (0.0.1):
     - Flutter
+  - path_provider (0.0.1):
+    - Flutter
 
 DEPENDENCIES:
   - Flutter (from `Flutter`)
+  - flutter_full_pdf_viewer (from `.symlinks/plugins/flutter_full_pdf_viewer/ios`)
```

Comment:
I have concern about this plugin:  flutter_full_pdf_viewer. To view the pdf, the plugin created its own Scaffold but  believe it's missing important components to modify the UI. I checked the plugin and it seems there's [issues](https://pub.dev/packages/flutter_full_pdf_viewer/score) in it. Have you tried other pdf viewer plug in?

Reply:
So, I haven't used any other plugins, as this is most suggested library to use while I was researching. If you look at the scores, you will see that that it doesn't support the web platform but it supports Android/iOS platform perfectly fine. For the statistical analysis, it says it has minor formatting file issue. But, they have provided the solution for it as well. It has 96% popularity as well. I think this will be fine. I also found other library like pdf viewer plugin but it has less popularity than this one. S oI preferred to use flutter_full_pdf_viewer. 

Also, I am not sure why it is running `- flutter_full_pdf_viewer (from `.symlinks/plugins/flutter_full_pdf_viewer/ios`)` in Podfile.lock file. Ans similar thing is happening with integration_test & path_provider dependencies as well. Check it out her

Reply:
flutter_full_pdf_viewer plugin is actually not bad. What I don't like is they limit the methods we can use. At the moment, PDFViewerScaffold only have the following:
1.) App bar customization
2.) The path
3.) A type bool called primary (used in appbar customization)

Problem with this is it do not accept child where we can add more widgets for body customization. As of now, user needs to scroll down to check other pages by using this plug in. How can we add UI where the user can simply click 'next' button to see other pages instead of scrolling down?

Reply:
@mansiPaTeLL I got your concern. There are so many pdf viewer plugging available to use. And it easy to use clark. for example, `https://pub.dev/packages/advance_pdf_viewer.` We are not sticking with particular plugin. If you want to use any other plugins you can free to use. I am just doing experimenting right now and we are not sticking with particular option. I hope this helps you. Thanks.

Reply:
I see, it's good to know that we're not stuck to this specific plugin

Reply:
@MummenRider Yes, but if you also look at the score for other plugins, they have less popularity so before using other plugins, we need to make sure it is compatible. But always if you find the better one, feel free to use it.